vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -100,6 +110,34 @@ int main(int argc, char** argv)
         l_rc = l_vpdToolObj.readKeyword(l_vpdPath, l_recordName, l_keywordName,
                                         l_isHardwareOperation, l_filePath);
     }
+    else if (!l_dumpObjFlag->empty())
+    {
+        if (!l_objectOption->empty())
```

Comment:
Should there be a log if dump object fails because of no object path passed?

Reply:
That is already taken care of by CLI11. `->needs(l_objectOption)`

Reply:
that is when it is not passed, what about empty case?

Reply:
Incase we give `vpd-tool --dumpObject -O `, we get following error message from CLI11:
`--object: 1 required TEXT missing`
`Run with --help for more information.`

Reply:
If you think this will never happen, please remove this.

Reply:
Addressed this. This `if (!l_objectOption->empty())` is not needed as Line 63 handles this.